pkg/sql/gcjob/gc_job.go

```diff
@@ -508,7 +508,7 @@ func shouldUseDelRange(
 ) bool {
 	// TODO(ajwerner): Adopt the DeleteRange protocol for tenant GC.
```

Comment:
>  Do we only replicate individual tenants

Yes, we only replicate individual tenants.

> Do we simply cut off the stream when the tenant is destroyed, or otherwise remove it from the target out-of-band?

If the tenant is dropped on the destination side, we stop the job. If the source side is dropped, I have a feeling we likely just keep going.  However, an active replication job keeps protected timestamps alive on both the source and destination side.  I believe the clear-range based protocol should wait out those protected timestamps before dropping the clears so it should be "OK" from that perspective.  We may want a DROP on the source side to kill the producer side streaming job that owns that protected timestamp to prevent holding up GC for too long. But I'm not sure that is essential.